src/main/java/duke/tasks/TaskList.java

```diff
@@ -0,0 +1,52 @@
+package duke.tasks;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    private ArrayList<Task> items;
+
+    public TaskList() {
```

Comment:
Maybe you can add in Javadocs here for the public methods

Reply:
Thank you for pointing it out. Will add in the javadocs here!